NitroxClient/MonoBehaviours/Multiplayer.cs

```diff
@@ -17,6 +17,7 @@ public class Multiplayer : MonoBehaviour
 
         public static Logic Logic;
         public static PacketSender PacketSender; //TODO: migrate logic out of and remove as member variable
+        public static Multiplayer main;
```

Comment:
We had a discussion about this and I didn't really know what to do. The resolution was that subnautica already uses plenty of singletons, but I do agree it is a bit silly.